ios/FluentUI.Demo/FluentUI.Demo/Demos/PersonaButtonCarouselDemoController.swift

```diff
@@ -0,0 +1,138 @@
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: PersonaButtonCarouselDemoController.smallBadgeReuseIdentifier)
+
+        tableView.separatorStyle = .none
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell: UITableViewCell
+        if showsFullCarousel(at: indexPath) {
+            fatalError("showsFullCarousel() should always return false right now!")
+        } else {
+            cell = self.tableView(tableView, badgeCellForRowAt: indexPath)
+        }
+
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch personaBadgeSize(for: section) {
+        case .small:
+            return "Small"
+        case .large:
+            return "Large"
+        }
+    }
+
+    // MARK: - Helpers
+
+    private func personaBadgeSize(for section: Int) -> MSFPersonaButtonSize {
+        return (section % 2 == 0) ? .small : .large
+    }
+
+    private func showsFullCarousel(at indexPath: IndexPath) -> Bool {
+        // TODO: Create PersonaButtonCarousel
+        return false
+    }
+
+    private func tableView(_ tableView: UITableView, badgeCellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell: UITableViewCell
+        let size = personaBadgeSize(for: indexPath.section)
+        switch size {
+        case .large:
+            cell = tableView.dequeueReusableCell(withIdentifier: PersonaButtonCarouselDemoController.largeBadgeReuseIdentifier, for: indexPath)
+        case .small:
+            cell = tableView.dequeueReusableCell(withIdentifier: PersonaButtonCarouselDemoController.smallBadgeReuseIdentifier, for: indexPath)
+        }
+
+        // Create the PersonaButton to be displayed
+        let personaBadge = MSFPersonaButtonView(size: size)
+        let persona = personas[indexPath.item]
+        personaBadge.state.image = persona.avatarImage
+        personaBadge.state.primaryText = persona.primaryText
+        personaBadge.state.secondaryText = persona.secondaryText
+        personaBadge.state.onTapAction = { [weak self, personaBadge] in
+            let alert = UIAlertController(title: nil, message: "PersonaButton tapped: \(personaBadge.state.primaryText ?? "(none)")", preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "OK", style: .default))
+            self?.present(alert, animated: true)
+        }
+
+        cell.contentView.addSubview(personaBadge.view)
+        cell.selectionStyle = .none
+        cell.backgroundColor = .clear
```

Comment:
Minor detail: can we set the background color with a different color so we can see the padding and the background of the PersonaButton?

Reply:
It should be visible already? The clear background passes through the light gray color of the table view.

Ah unless you're mostly using dark mode, where they're both black. Okay, I'll give it a little *pop*